sdk/src/main/java/com/mapbox/maps/MapboxMap.kt

```diff
@@ -261,6 +261,20 @@ class MapboxMap internal constructor(
     return nativeMapWeakRef.call { this.resourceOptions }
   }
 
+  /**
+   * Clears temporary map data.
+   *
+   * Clears temporary map data from the data path defined in the given resource options.
+   * Useful to reduce the disk usage or in case the disk cache contains invalid data.
+   * Note that calling this API will affect all maps that use the same data path.
+   * Note that calling this API does not affect persistent map data like offline style packages.
+   *
+   * @param callback Called once the request is complete or an error occurred.
+   */
+  fun clearData(callback: AsyncOperationResultCallback) {
```

Comment:
Perhaps I've misunderstood a bit - but why do we need this method if we have a static one? Just for convenience?

Reply:
yes, an instance method is more discoverable, and doesn't have to provide resource options. The static method is more used for offline cases, where a MapboxMap instance doesn't necessarily exist.